src/PlusDataCollection/Atracsys/vtkPlusAtracsysTracker.cxx

```diff
@@ -0,0 +1,396 @@
+{
+  LOG_TRACE("vtkPlusAtracsysTracker::Probe");
+  return PLUS_SUCCESS;
+}
+
+//----------------------------------------------------------------------------
+PlusStatus vtkPlusAtracsysTracker::InternalConnect()
+{
+  LOG_TRACE("vtkPlusAtracsysTracker::InternalConnect");
+
+  ATRACSYS_RESULT result;
+
+  // Connect to tracker
+  result = this->Internal->Tracker->Connect();
+  if (result != ATR_SUCCESS && result != AtracsysTracker::ATRACSYS_RESULT::WARNING_CONNECTED_IN_USB2)
+  {
+    LOG_ERROR(this->Internal->Tracker->ResultToString(result));
+    return PLUS_FAIL;
+  }
+  else if (result == AtracsysTracker::ATRACSYS_RESULT::WARNING_CONNECTED_IN_USB2)
+  {
+    LOG_WARNING(this->Internal->Tracker->ResultToString(result));
+  }
+
+  // get device type
+  this->Internal->Tracker->GetDeviceType(this->Internal->DeviceType);
+
+  // if spryTrack, setup for onboard processing and disable extraneous marker info streaming
+  if (this->Internal->DeviceType == AtracsysTracker::DEVICE_TYPE::SPRYTRACK_180)
+  {
+    bool succeeded = true;
+    succeeded = succeeded && (this->Internal->Tracker->EnableOnboardProcessing(true) == ATR_SUCCESS);
+    succeeded = succeeded && (this->Internal->Tracker->EnableImageStreaming(false) == ATR_SUCCESS);
+    succeeded = succeeded && (this->Internal->Tracker->EnableWirelessMarkerStatusStreaming(false) == ATR_SUCCESS);
+    succeeded = succeeded && (this->Internal->Tracker->EnableWirelessMarkerBatteryStreaming(false) == ATR_SUCCESS);
+    if (!succeeded)
+    {
+      LOG_WARNING("Failed to setup spryTrack for onboard processing.");
+    }
+  }
+
+  // load passive geometries onto Atracsys
+  std::map<std::string, std::string>::iterator it;
+  for (it = begin(this->Internal->PlusIdMappedToGeometryFilename); it != end(this->Internal->PlusIdMappedToGeometryFilename); it++)
+  {
+    // load user defined geometry file
+    // TODO: add check for conflicting marker IDs
+    std::string geomFilePath = vtkPlusConfig::GetInstance()->GetDeviceSetConfigurationPath(it->second);
+    int geometryId;
+    if ((result = this->Internal->Tracker->LoadMarkerGeometry(geomFilePath, geometryId)) != ATR_SUCCESS)
+    {
+      LOG_ERROR(this->Internal->Tracker->ResultToString(result));
+      return PLUS_FAIL;
+    }
+    std::pair<int, std::string> newTool(geometryId, it->first);
+    this->Internal->FtkGeometryIdMappedToToolId.insert(newTool);
+  }
+
+  if ((result = this->Internal->Tracker->SetMaxMissingFiducials(this->Internal->MaxMissingFiducials)) != ATR_SUCCESS)
+  {
+    LOG_WARNING(this->Internal->Tracker->ResultToString(result));
+  }
+
+  // make LED blue during pairing
+  this->Internal->Tracker->SetUserLEDState(0, 0, 255, 0);
+
+  // pair active markers
+  if ((result = this->Internal->Tracker->EnableWirelessMarkerPairing(true)) != ATR_SUCCESS)
+  {
+    LOG_ERROR(this->Internal->Tracker->ResultToString(result));
+    return PLUS_FAIL;
+  }
+  LOG_INFO(endl << " *** Put active markers in front of the tracker to pair ***" << endl);
```

Comment:
Log messages are not really for displaying instructions to users. Change it to LOG_INFO("Active marker pairing period started"). Do the same for end of marker pairing period. Instructions for users can be implemented at the application level or described in user manual.